doc/conf.py

```diff
@@ -378,7 +379,8 @@ def gallery_image_warning_filter(record):
 formats = {'html': ('png', 100), 'latex': ('pdf', 100)}
 plot_formats = [formats[target] for target in ['html', 'latex']
                 if target in sys.argv] or list(formats.values())
-
+# make 2x images for srcset argument to <img>
+plot_srcset = '2x'
```

Comment:
Actually, now I remember why I did this.  You can't pass lists in to sphinx-build, so comma-delimited strings seemed the way t go?  But I don't feel strongly either way

Reply:
It will work, and is [passed as a comma-delimited string](https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-D) already. But what's missing here that's in `mathmpl` is that the default should be `[]` so Sphinx knows to turn it into a list.